shared/constants/chat2/message.js

```diff
@@ -356,7 +356,7 @@ const makeMessageSystemChangeRetention: I.RecordFactory<MessageTypes._MessageSys
     isInherit: false,
     isTeam: false,
     membersType: 0,
-    policy: RPCChatTypes.RetentionPolicy,
```

Comment:
this isn't a value so it was null!

Reply:
https://user-images.githubusercontent.com/86695676/123932968-da3b5300-d989-11eb-8b55-eef21c1c964e.mp4

Reply:
Is that good